mozilla_django_oidc/auth.py

```diff
@@ -110,8 +110,8 @@ def verify_token(self, token, **kwargs):
 
         # Verify the token
         verified_token = self._verify_jws(
-            token,
-            key,
+            smart_bytes(token),
+            smart_bytes(key),
```

Comment:
Do you think this will be called with Django promises? If not, I'd take `django.utils.encoding.force_bytes` instead.